### DISCUSSION TITLE
Is possible to append baseURL to public image directory?

### DISCUSSION THREAD

**Discussion by maico910** [nuxt/nuxt#16569] [Questions]
I has been set `baseUrl` like this:
```typescript
// nuxt.config.ts
export default defineNuxtConfig({
  app: {
    baseURL: "/custom-url/"
  }
}
```

I want when using static image, on build generate the path with baseURL:

```HTML
// generated path
<img src="/my-image.png">

// wanted path
<img src="/custom-url/my-image.png">
```

In Nuxt 2, I finded this documention, on build will add automatically by default static assets prefix mentioned in [link](https://nuxtjs.org/docs/directory-structure/static#static-asset-prefix) and this issue asking for this feature in this [issue](https://github.com/nuxt/nuxt.js/issues/2753)

This is my nuxt.config.ts but it's not working:
```typescript
export default defineNuxtConfig({
  ssr: true,
  app: {
    baseURL: "/custom-url/"
  },
  render: {
    static: {
      prefix: true,
    },
  },
  router: {
    base: "/custom-url/",
  },
  // (...)
}
```

Does I'm missing something or exist this feature on Nuxt 3?

**Comment by thiswallz** [comment]
have you try the root path, instead of absolute or relative path? like:

``` html
<img src="~/my-image.png" />
```

**Reply by maico910** [reply]
image path started with `~` is for processed image by build tools, this is not my case

**Reply by thiswallz** [reply]
@maico910  got it it seems you have some extra slash at the beginning, just use:

``` html
<img src="my-image.png">
```

otherwise the  leading `/ ` will overwrite the router base config

**Reply by maico910** [reply]
from the [doc Nuxt 2](https://nuxtjs.org/docs/directory-structure/static/#static-assets) and [doc Nuxt 3](https://v3.nuxtjs.org/getting-started/assets#public-directory) is necessary leading by `/` to access these files
